scripts/ci/install_and_check_minimum_requirements.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -ex
```

Comment:
```suggestion
#!/usr/bin/env bash
set -euo pipefail
```

We should really have `set -euo pipefail` by default in every script and only do less if it actually interferes in some way.